source/includes/fact-cdc-recoverable.rst

```diff
@@ -0,0 +1,2 @@
+You can recover a continuous sync job within 24-hours of failure. 
```

Comment:
(nit -- wording)
24-hour is hyphenated when it's an adjective. In this case, it's a noun. 
So:
- 24-hour period
- within 24 hours of failure
```suggestion
You can recover a continuous sync job within 24 hours of failure. 
```